cypress/integration/walkthrough.js

```diff
@@ -0,0 +1,41 @@
+// walkthrough.js created with Cypress
+//
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+
+describe("Walkthrough Tests", () => {
+  it("should load data for each dot", () => {
+    cy.visit("/");
+
+    cy.get("h2").should("contain.text", "Learn and build in web3.");
+
+    cy.get(".js-dot0").click();
```

Comment:
looks like the tests need to be updated 
![image](https://user-images.githubusercontent.com/1197406/155801590-bfc54e56-1d54-4a03-a345-8baaf34787a7.png)

Reply:
Fixed